yoti/Readme.md

```diff
@@ -0,0 +1,156 @@
+# Yoti
+
+Welcome to the Yoti Drupal SDK. This repo contains the tools you need to
+quickly integrate your Drupal back-end with Yoti, so that your users can
+share their identity details with your application in a secure and trusted
+way.
+
+## Table of Contents
+
+1) [An Architectural view](#an-architectural-view) -
+High level overview of integration
+
+2) [References](#references) -
+Guides before you start
+
+3) [Installing the SDK](#installing-the-sdk) -
+How to install our SDK
+
+4) [Plugin Setup](#plugin-setup)-
+How to set up the plugin in Drupal
+
+5) [Linking existing accounts to use Yoti authentication
+](#linking-existing-accounts-to-use-yoti-authentication)
+
```

Comment:
Just the same section link here that I mentioned in https://github.com/getyoti/yoti-drupal/pull/38

Reply:
Fixed in fd1ea74